lib/objects.coffee

```diff
@@ -44,11 +44,13 @@ Meteor.methods
           pts: [xywType]
           color: String
           width: Number
+          opacity: Number
```

Comment:
These should probably be made optional in the same way as `fill`.

Reply:
I think I did it right but I basically aped some other code